test/integration/cloud-config-vmware.ini.template

```diff
@@ -0,0 +1,7 @@
+[DEFAULT]
```

Comment:
We only want a single cloud plugin for vmware/vcenter, so any config should go in the existing `cloud-config-vcenter.ini.template` file instead.